nextflow.config

```diff
@@ -16,8 +16,11 @@ params {
     fasta_fai                  = null
     fasta_dict                 = null
     fasta_mapperindexdir       = null
+    fasta_circular_target      = null
+    fasta_mitochondrion_header = null
     fasta_largeref             = false
 
+
```

Comment:
```suggestion
```

Reply:
Please also add a line between 27 and 28 before `//MultiQC options` and remove lines 50 and 59.